app/src/main/res/layout/view_user_profile.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:paddingTop="@dimen/profile_padding_vertical_default"
+    android:paddingStart="@dimen/profile_padding_horizontal_default"
+    android:paddingEnd="@dimen/profile_padding_horizontal_default"
+    android:paddingBottom="@dimen/profile_padding_vertical_default"
+    android:dividerPadding="6dp">
+
+    <Spinner
+        android:id="@+id/sex_spinner"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:hint="@string/profile_sex_prompt"/>
+
+    <EditText
+        android:id="@+id/birthday_edit_text"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:editable="false"
+        android:hint="@string/profile_birthday_prompt"/>
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal" >
+        <EditText
+            android:id="@+id/height_feet_field"
+            android:layout_width="0dp"
+            android:layout_weight=".5"
+            android:layout_height="wrap_content"
+            android:singleLine="true"
+            android:inputType="number"
+            android:hint="@string/profile_height_prompt"/>
+        <EditText
+            android:id="@+id/height_inches_field"
+            android:layout_width="0dp"
+            android:layout_weight=".5"
+            android:layout_height="wrap_content"
+            android:singleLine="true"
+            android:inputType="number"
+            android:hint="@string/profile_height_inches_prompt"/>
+    </LinearLayout>
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        android:paddingStart="@dimen/profile_padding_horizontal_labels"
+        android:paddingEnd="@dimen/profile_padding_horizontal_labels" >
+        <TextView
+            android:textSize="@dimen/profile_label_text_size"
+            android:layout_width="0dp"
+            android:layout_weight=".5"
+            android:layout_height="wrap_content"
+            android:singleLine="true"
+            android:text="@string/profile_height_feet_units"
+            android:labelFor="@+id/height_feet_field" />
+        <TextView
+            android:textSize="@dimen/profile_label_text_size"
+            android:layout_width="0dp"
+            android:layout_weight=".5"
+            android:layout_height="wrap_content"
+            android:paddingStart="@dimen/profile_padding_horizontal_labels"
+            android:singleLine="true"
```

Comment:
I think this is deprecated? I'm pretty sure `maxLines=1` is preferred. Also be sure to add `ellipsize="end"`.

Reply:
Oh wait, is this for the text underneath the text field? Look up `TextInputLayout` I think it's called.